user_tools/src/spark_rapids_pytools/wrappers/dataproc_gke_wrapper.py

```diff
@@ -0,0 +1,142 @@
+               It accepts one of the following ("CLUSTER", "JOB" and the default value "MATCH").
+                "MATCH": keep GPU cluster same number of nodes as CPU cluster;
+                "CLUSTER": recommend optimal GPU cluster by cost for entire cluster;
+                "JOB": recommend optimal GPU cluster by cost per job
+        :param jvm_heap_size: The maximum heap size of the JVM in gigabytes
+        :param verbose: True or False to enable verbosity to the wrapper script
+        :param cpu_discount: A percent discount for the cpu cluster cost in the form of an integer value
+                (e.g. 30 for 30% discount).
+        :param gpu_discount: A percent discount for the gpu cluster cost in the form of an integer value
+                (e.g. 30 for 30% discount).
+        :param global_discount: A percent discount for both the cpu and gpu cluster costs in the form of an
+                integer value (e.g. 30 for 30% discount).
+        :param rapids_options: A list of valid Qualification tool options.
+                Note that the wrapper ignores ["output-directory", "platform"] flags, and it does not support
+                multiple "spark-property" arguments.
+                For more details on Qualification tool options, please visit
+                https://nvidia.github.io/spark-rapids/docs/spark-qualification-tool.html#qualification-tool-options
+        """
+        if verbose:
+            # when debug is set to true set it in the environment.
+            ToolLogging.enable_debug_mode()
+        wrapper_qual_options = {
+            'platformOpts': {
+                'credentialFile': credentials_file,
+                'deployMode': DeployMode.LOCAL,
+            },
+            'migrationClustersProps': {
+                'cpuCluster': cpu_cluster,
+                'gpuCluster': gpu_cluster
+            },
+            'jobSubmissionProps': {
+                'remoteFolder': remote_folder,
+                'platformArgs': {
+                    'jvmMaxHeapSize': jvm_heap_size
+                }
+            },
+            'eventlogs': eventlogs,
+            'filterApps': filter_apps,
+            'toolsJar': tools_jar,
+            'gpuClusterRecommendation': gpu_cluster_recommendation,
+            'cpuDiscount': cpu_discount,
+            'gpuDiscount': gpu_discount,
+            'globalDiscount': global_discount
+        }
+
+        tool_obj = QualificationAsLocal(platform_type=CspEnv.DATAPROC_GKE,
+                                        output_folder=local_folder,
+                                        wrapper_options=wrapper_qual_options,
+                                        rapids_options=rapids_options)
+        tool_obj.launch()
+
+
+class DataprocGKEWrapper:  # pylint: disable=too-few-public-methods
+    """
+    A wrapper script to run RAPIDS Accelerator tools (Qualification, Profiling, and Bootstrap) on Gcloud Dataproc GKE.
```

Comment:
Nit: Profiling and Bootstrap can be removed since this PR adds only qualification support.